DNN Platform/Library/Data/DataProvider.cs

```diff
@@ -2407,7 +2412,7 @@ public virtual void RestoreUser(int userId, int portalId)
                                       isApproved,
                                       refreshRoles,
                                       lastIpAddress,
-                                      passwordResetToken,
+                                      GetNull(passwordResetToken),
```

Comment:
Was this a "bug fix" along the way?

Reply:
Looks like it to me. I tried to update using Null.NullGuid and the field was not NULL. So IMO that is not the behaviour one expects. From what I can tell no code actually tries to clear the reset token. There is no mechanism for this. All code just lets it expire. I wanted to NULL it so no open vectors remained to an account.

Reply:
are you able to pass database Null instead?

Reply:
The default value of this column is NULL in SQL. Without this change I can never reset to that value. But with this change the column will be NULL again. So I guess this is actually exactly what you want Sebastian.